src/PrintOrder/ComponentToPrint/ComponentToPrint.css

```diff
@@ -0,0 +1,7 @@
+.colB {
```

Comment:
can we use more suitable name? I used just to not spend time
mb `orderLineDetails`?